www/tuma02/sp2/assets/js/main.js

```diff
@@ -0,0 +1,216 @@
+//definition of used elements
+const searchRecord = $('#search-record');
+const cocktailInput = $('#search-record [name="cocktail-input"]');
+const searchIngredient = $('#search-ingredient');
+const ingredientInput = $('#search-ingredient [name="ingredient-input"]');
+const tooltipAll = $('#tooltip-all');
+const cannotFind1 = $('#cannot-find1');
+const cannotFind2 = $('#cannot-find2');
+const spinner = $(`<div class="lds-hourglass"></div>`);
+const time = $('.time');
+const recept = $('.recept');
+const cocktailTitle = $('#cocktail-title');
+const measuresIngredients = $('#measures-ingredients');
+const instruction = $('#instruction');
+const drinks = $('#drinks');
+const info = $('#info');
+const tableWithDrinks = $('.table-layout');
+
+//definition of helping arrays
+var date;
+const array = [];
+var cocktail = [];

```

Comment:
const

Reply:
var date;
const cocktail = [];

done

Reply:
let date;
var cocktail;

při constu to nepracuje, ale ted cocktail není array.
done